### PR TITLE
Grammar update reference-article.txt

### PR DESCRIPTION
**Pull request by luxuryluke**
Small changes to grammar and code spacing

### CODE REVIEW

File: content/docs/3_reference/3_panel/3_fields/0_color/reference-article.txt [hunk 1]
```diff
@@ -94,7 +94,7 @@ color:
 <since v="4.1.0">
 ### Dynamic options
 
-Our options (link: docs/guide/blueprints/query-language text: query syntax) offers a very powerful way of to add an automatically generated option lists to a color field.
+Our options (link: docs/guide/blueprints/query-language text: query syntax) offer a powerful way of adding automatically-generated option lists to a color field.
```
Comment (distantnative) [inline content/docs/3_reference/3_panel/3_fields/0_color/reference-article.txt]: Is this right? `syntax` is singular and so shouldn't it be `offers`? `automatically generated` should also not have a hyphen, it's not a composable, is it?

Comment (texnixe) [reply content/docs/3_reference/3_panel/3_fields/0_color/reference-article.txt]: The hyphen is fine, I think, but `offer` when referring to syntax seems strange. But the options are a bit alone there, so I'd suggest:
```
Our (link: docs/guide/blueprints/query-language text: options query syntax) offers ...
```

Comment (distantnative) [reply content/docs/3_reference/3_panel/3_fields/0_color/reference-article.txt]: I can't find many mentions of `automatically-generated` on the web. It's either `auto-generated` or `automatically generated`. E.g. https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

Comment (luxuryluke) [reply content/docs/3_reference/3_panel/3_fields/0_color/reference-article.txt]: Correct about 'offer/offers', misread it in code mode.

**Review by luxuryluke — COMMENTED** [review]
removed automatically hyphen.

**Review by luxuryluke — COMMENTED** [review]
reverted to the singular 'offer'

**Comment by texnixe** [comment]
Thanks again!

**Pull request merged**